app/vmalert/notifier/init.go

```diff
@@ -15,18 +15,37 @@ var (
 	configPath                    = flag.String("notifier.config", "", "Path to configuration file for notifiers")
 	suppressDuplicateTargetErrors = flag.Bool("notifier.suppressDuplicateTargetErrors", false, "Whether to suppress 'duplicate target' errors during discovery")
 
-	addrs                 = flagutil.NewArray("notifier.url", "Prometheus alertmanager URL, e.g. http://127.0.0.1:9093")
+	addrs = flagutil.NewArray("notifier.url", "Prometheus alertmanager URL, e.g. http://127.0.0.1:9093")
+
 	basicAuthUsername     = flagutil.NewArray("notifier.basicAuth.username", "Optional basic auth username for -notifier.url")
 	basicAuthPassword     = flagutil.NewArray("notifier.basicAuth.password", "Optional basic auth password for -notifier.url")
 	basicAuthPasswordFile = flagutil.NewArray("notifier.basicAuth.passwordFile", "Optional path to basic auth password file for -notifier.url")
 
+	authorizationType            = flagutil.NewArray("notifier.authorization.type", "Optional authorization type (`basic or bearer`) for -notifier.url")
+	authorizationCredentials     = flagutil.NewArray("notifier.authorization.credentials", "Optional authorization credentials for -notifier.url")
+	authorizationCredentialsFile = flagutil.NewArray("notifier.authorization.credentialsFile", "Optional path to authorization credentials file for -notifier.url")
```

Comment:
I think we can ignore `authorization` options here and in the rest of packages from this PR.
Looks like we don't use it in vmagent as well, except of auth options configured via promscrape file. 
@f41gh7 is my understanding correct?

Reply:
Yep, i think there is no need in these options. Bearer token and basic auth should be fine.

Reply:
removed